MonstieWash/Assets/Scripts/MonsterAi/MonsterBrain.cs

```diff
@@ -0,0 +1,308 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class MonsterBrain : MonoBehaviour
+{
+
+    [Tooltip("Add all moodtype objects intended for this brain here.")] [SerializeField] private List<MoodType> moodData; //Scriptable objects holding data about moods.
+
+    private Dictionary<int,float> activeMoods; //Current moods status. int refers to id and number of mood in list, float refers to current value of mood on its own scale.
+
+    private Dictionary<string, int> activeMoodNames; // Current moods and their names. int refers to id and number of mood in list, string refers to name.
+
+    [Tooltip("Updates the debug window when turned on")] [SerializeField] public bool Debug;
+
+
+    [Tooltip("Pauses the brain when on.")][SerializeField] public bool Pause;
+
+    private int designerSanityBuff; // A multiplier to reduce the tiny size of numbers used in setting up scriptable objects. Recommended set at 10.
+
+
+    [Tooltip("Attach Text Mesh Pro Box here for displaying debug info")][SerializeField] public TextMeshProUGUI DebugUi;
+
+    [HideInInspector] public Dictionary<int, float> ActiveMoods { get { return activeMoods; } }
+    [HideInInspector] public List<MoodType> MoodData { get { return moodData; } }
+
+
+    private void Awake()
+    {
+        designerSanityBuff = 10;
+        activeMoods = new Dictionary<int, float>();
+        activeMoodNames = new Dictionary<string, int>();
+
+        loadMoods();
+    }
+
+
+    private void Update()
+    {
+        if (Pause) return;
+
+        //Moods move by their natural change value to their resting point value.
+        naturalChange();
+        //Moods are affected by Chaos Multipliers.
+        chaoticInterference();
+        //Moods affect their positive chain reactions.
+        positiveChainReactions();
+        //Moods affect their negative chain reactions.
+        negativeChainReactions();
+        //Moods are kept to their upper and lower limits.
+        maintainLimits();
+
+
+        //Debug Updates
+        if (Debug) updateDebugText();
+    }
+
+
+    /// <summary>
+    /// Accesses the Moods resting point and natural rate of change, then moves mood towards its resting point by its rate of change.
+    /// </summary>
+    private void naturalChange()
```

Comment:
This goes for ALL the methods, they should be named using PascalCase and not camelCase. This means all methods should start with capital letters. I've only selected this one for reference but you'll need to change all of them to follow this convention.
```suggestion
    private void NaturalChange()
```